boards/stm32f4discovery/include/periph_conf.h

```diff
@@ -211,6 +211,41 @@ static const i2c_conf_t i2c_config[] = {
 #define I2C_NUMOF           (sizeof(i2c_config) / sizeof(i2c_config[0]))
 /** @} */
 
+/**
```

Comment:
I know it was done this way in the initial PR, but I'm against having an eth peripheral configured by default on a board that doesn't provide an Ethernet interface.
I would prefer to have it on the nucleo-f429zi for F4 CPUs.

Reply:
Maybe I should also check if there other boards from the 4/7 family which got Ethernet. Should work for other boards from the same family as well.

Reply:
Removed by now since the DMA interface changed and I've got no test board for the f4 family. If somebody have a board to test i will add this again in the correct file